### ISSUE TITLE
Add a LICENSE file?

### ISSUE THREAD

**Issue by Trott** [nodejs/Release#721]
I was asked today how to get permission to use the image in the README. I went to see what the content is licensed with and there doesn't seem to be an indication.

**Comment by targos** [comment]
What kind of license should we add? Something Creative Commons since there is no code in this repo?

**Comment by Trott** [comment]
> What kind of license should we add? Something Creative Commons since there is no code in this repo?

Yeah, I imagine we'd need to consult with the Foundation (and I can bring this to the CPC), but [CC0](https://creativecommons.org/share-your-work/public-domain/cc0/) would be great by me. If we want to restrict people's ability to alter the image (so they don't alter it to include the wrong dates or something), there's [CC-BY-ND](https://creativecommons.org/licenses/by-nd/4.0/).

And for more options: https://creativecommons.org/about/cclicenses/

**Comment by richardlau** [comment]
If it helps at all with deciding what license, we generate the image using https://github.com/nodejs/lts-schedule (which has its own license ambiguity https://github.com/nodejs/lts-schedule/issues/13) based on the schedule.json data in this repository.

**Comment by mhdawson** [comment]
Foundation guidance on licences - https://github.com/openjs-foundation/cross-project-council/blob/main/IP_POLICY_GUIDANCE.md#1-licensing

CC BY 4.0
MIT

are the two listed for documentation

**Comment by targos** [comment]
@mhdawson the link is now broken.
